src/pages/cli/migration/override.mdx

```diff
@@ -1,16 +1,16 @@
 export const meta = {
   title: `Migration to enabled override feature`,
-  description: `Upgrading to Amplify CLI version 7 and above with a project created prior requires an migration to enable the new "override" capability.`,
+  description: `Upgrading to Amplify CLI version 7 and above with a project created prior requires an migration to enable the new "override" capability.`
```

Comment:
should be "requires a migration" ?

Reply:
you are absolutely correct I did not notice that

Reply:
Thanks!